crates/esthri-test/src/lib.rs

```diff
@@ -30,6 +31,10 @@ pub fn test_data(name: &str) -> PathBuf {
     test_data_dir().join(name)
 }
 
+pub fn randomised_name(name: &str) -> String {
+    format!("{}-{}", Uuid::new_v4(), name)
```

Comment:
We should probably add a default age out to the esthri-test bucket now, but it'll need to be done a little carefully as some of the download tests rely on files being in the right place, so these shouldn't be deleted. Probably not super high priority as the integration tests don't get run super regularly and don't upload too much, so have created a story to track it: https://swift-nav.atlassian.net/browse/DEVINFRA-630